console-ui/src/globalLib.js

```diff
@@ -497,13 +497,13 @@ const request = (function(_global) {
     try {
       token = JSON.parse(localStorage.token);
     } catch (e) {
-      console.log('Token Erro', localStorage.token, e);
+      console.log('Token Error', localStorage.token, e);
```

Comment:
removing the console.log will be better.

Reply:
I think that we should preserve it util finishing refactor the logger system of the frontend for debugging,  and there are many similar issues in the frontend of nacos.